src/App.js

```diff
@@ -1,26 +1,9 @@
-import logo from './logo.svg';
 import './App.css';
+import Calculator from './components/calculator';
 
 function App() {
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit
-          <code> src/App.js </code>
-          and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
+    <Calculator />
   );
```

Comment:
- [ ] [REQUIRED] Kindly ensure that the `App.js`  component is class-based in order to properly follow the project requirements.

Reply:
You're right @Whoistolu! Thank you so much for the reminder, I totally missed that, I'll start fixing it.